src/main/java/com/lilybookclub/controller/AuthController.java

```diff
@@ -0,0 +1,36 @@
+package com.lilybookclub.controller;
+
+import com.lilybookclub.dto.request.LoginRequest;
+import com.lilybookclub.dto.request.SignUpRequest;
+import com.lilybookclub.dto.response.LoginResponse;
+import com.lilybookclub.dto.response.SignUpResponse;
+import com.lilybookclub.service.AuthService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RequestMapping("/api/v1/auth")
+@RestController
+@RequiredArgsConstructor
+public class AuthController {
+
+    private final AuthService authService;
+
+    @PostMapping("/signup")
```

Comment:
No need for the "/signup". Also if this creates a user, it should be in the user entity